src/dos/drive_local.cpp

```diff
@@ -46,11 +46,22 @@
 #include "cross.h"
 #include "inout.h"
 
+bool localDrive::FileIsReadOnly(char* name)
+{
+	if (isReadOnly()) {
```

Comment:
Can we not carry forward old conventions? So `IsReadOnly()` perhaps?

Reply:
Renamed.  Also renamed `isRemovable` and `isRemote` for consistency.